sdk/test-utils/recorder-new/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## 1.0.0 (Unreleased)
 
+## 2021-11-08
+
+- Allows storing the dynamically created variables in the record mode. Recorder registers the variables as part of the recording. Using the "variables" in playback mode would give the key-value pairs that are stored in record mode.
+
+  Example:
+
+  ```ts
+  if (!isPlaybackMode()) {
+    recorder.variables["random-1"] = `random-${Math.ceil(Math.random() * 1000 + 1000)}`;
+  }
+  ```
+
+  Use this `recorder.variables["random-1"]` whereever you'd like to use in your test after the if-block. (This would work in all three modes - record/playback/live just by adding the if-block above)
```

Comment:
```suggestion
  Use `recorder.variables["random-1"]` to access the value of the variable after setting it. The variable can be accessed in all three modes -- record, playback, and live -- as long as it is set in record mode before it is accessed.
```

I'm assuming that this is the right interpretation of how the "variable" system works.